Unreal/CarlaUnreal/Plugins/Carla/Source/Carla/Settings/CarlaSettingsDelegate.cpp

```diff
@@ -48,7 +48,7 @@ void UCarlaSettingsDelegate::RegisterSpawnHandler(UWorld *InWorld)
 void UCarlaSettingsDelegate::OnActorSpawned(AActor *InActor)
 {
   check(CarlaSettings != nullptr);
-  if (InActor != nullptr && IsValid(InActor) && !InActor->IsPendingKill() &&
+  if (InActor != nullptr && IsValid(InActor) &&
```

Comment:
```suggestion
  if (IsValid(InActor) &&
```

IsValid() already checks if InActor is null.